app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if local_assigns[:model].errors.any? %>
+  <div class="alert alert-danger" role="alert">
+    <h2 class="alert__heading">There's a problem</h2>
+    <ul>
+      <% local_assigns[:model].errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
```

Comment:
The partial actually only needs the errors, not the whole model (although that might be an implementation detail, your call)